platform/source/mbed_retarget.cpp

```diff
@@ -404,17 +488,24 @@ int mbed::bind_to_fd(FileHandle *fh)
     stdio_out_prev[fildes] = 0;
 
     return fildes;
+#else
+    return -1;
+#endif // !(MBED_CONF_PLATFORM_STDIO_MINIMAL_CONSOLE_ONLY)
 }
 
 static int unbind_from_fd(int fd, FileHandle *fh)
 {
+#if !(MBED_CONF_PLATFORM_STDIO_MINIMAL_CONSOLE_ONLY)
```

Comment:
Static function used in fdopen.
It will be automatically removed if fdopen is compiled out.

Reply:
https://github.com/hugueskamba/mbed-os/pull/4#discussion_r339448144